helm/templates/server/configmap.yaml

```diff
@@ -26,4 +26,11 @@ data:
     pinot.server.segment.fetcher.protocols=file,http,gs
     pinot.server.segment.fetcher.gs.class=org.apache.pinot.common.utils.fetcher.PinotFSSegmentFetcher
     {{- end }}
+    {{- if eq .Values.cluster.storage.scheme "s3" }}
+    pinot.server.storage.factory.class.s3=org.apache.pinot.plugin.filesystem.S3PinotFS
+    pinot.server.storage.factory.s3.region={{ .Values.cluster.storage.s3.region }}
+    pinot.server.segment.fetcher.protocols=file,http,s3
+    pinot.server.segment.fetcher.s3.class=org.apache.pinot.common.utils.fetcher.PinotFSSegmentFetcher
+    pinot.controller.storage.factory.s3.disableAcl=false
```

Comment:
What is this for? Intentional?

Reply:
This option is to grant full control to bucket owner.